### ISSUE TITLE
Qwen2-vl training performance not good

### ISSUE THREAD

**Issue by ZiangWu-77** [alibaba/Pai-Megatron-Patch#647]
<img width="1289" height="444" alt="Image" src="https://github.com/user-attachments/assets/9d0757f4-01a9-4a3b-9cea-5eac8e0984b8" />
Great work. I used the patch to sft qwen2-vl, and found it worked fine. 
I have aligned setting e.g. the mean/std of image and lr/max length. However the loss is about 0.2 higher than other framework e.g. (ms-swift and llamafactory) from the begining when using same finetuning dataset Cambrian737K. Also the final sft model's performance was about 3 points lower than ms-swift or llamafactory.  
Maybe someone know why it happens? Thx for reply.
